### PR TITLE
I add new loss function for using object detection task.

### PR DESCRIPTION
**Pull request by hisushanta**
This loss function is for using a lot of the time object detection tasks.
Please see what look like my loss function : 
Screenshot from 2023-07-10 19-49-09

### CODE REVIEW

**Comment by hi-sushanta** [comment]
please can you tell me how to solve EasyCLA authentication?

**Comment by hi-sushanta** [comment]
And also sorry for I again submit a pull request.

**Comment by albanD** [comment]
Hey!
I would suggest waiting on a decision in https://github.com/pytorch/pytorch/issues/104711 if we want to add it or not before working on making the PR ready for merging!

**Comment by hi-sushanta** [comment]
My main question is how to solve this issue: 
Screenshot from 2023-07-10 20-21-56

**Comment by hi-sushanta** [comment]
[davidradl](https://github.com/davidradl) say as for me please:
@hi-sushanta you need to sort out your CLA. 

That is the reason I say how to solve this issue: EasyCLA issue.

**Comment by albanD** [comment]
You can click to the easycla image at the top of the comment: https://github.com/pytorch/pytorch/pull/104874#issuecomment-1629080312
link is [here](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/29311126/65600975/104874/#/?version=2) as well if you need.
You can then follow the instructions there to sign the CLA.

**Comment by hi-sushanta** [comment]
I submit already this file but not solve this problem.

**Comment by albanD** [comment]
Hi,

As you can see in the CLA error comment above: https://github.com/pytorch/pytorch/pull/104874#issuecomment-1629080312
The problem is that the email you used to sign the commit is not registered to this github account. You just need to add this email to your github account.

**Comment by hi-sushanta** [comment]
A few days ago change my previous mail hiwhy@proton.me  -> iamchi@skiff.com. I think this is a problem.

**Comment by hi-sushanta** [comment]
@cla-bot check : cla bot label

**Comment by hi-sushanta** [comment]
Hi, I solve my CLA problem. So now you can see my code and review it.

**Comment by hi-sushanta** [comment]
If anyone can review my code?

**Comment by hi-sushanta** [comment]
How many years take to review this code?

**Comment by janeyx99** [comment]
hi @hi-sushanta, our process in getting code reviewed is for the feature to be approved first. Thus, this PR will likely not get any review until the feature in https://github.com/pytorch/pytorch/issues/104711 gets approved.

**Comment by hi-sushanta** [comment]
how many days take to approve my loss function?

**Comment by mikaylagawarecki** [comment]
Hey, we will not be accepting this loss function at this point in time as there hasn't been sufficient justification on the issue thus far.
